packages/server/graphql/types/CreatePollInput.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  GraphQLFloat,
+  GraphQLID,
+  GraphQLInputObjectType,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLString
+} from 'graphql'
+import PollOptionInput from './CreatePollOptionInput'
+
+const CreatePollInput = new GraphQLInputObjectType({
+  name: 'CreatePollInput',
+  fields: () => ({
+    teamId: {
+      type: GraphQLNonNull(GraphQLID),
+      description: 'Foreign key for the team this poll belongs to'
+    },
+    discussionId: {
+      type: GraphQLNonNull(GraphQLID),
+      description: 'Foreign key for the discussion this was created in'
+    },
+    threadSortOrder: {
+      type: GraphQLNonNull(GraphQLFloat),
+      description: 'The order of this threadable, relative to threadParentId'
```

Comment:
polls don't have a `threadParentId` because they can't be a "reply" to another type, right?

Reply:
Oh yeah. The comment should not say `relative to threadParentId`. Just to make sure, `threadSortOrder` is still a valid property used to determine the sort order in the discussion, or am I thinking wrong here? 🙏